src/Query/TaskGroups.ts

```diff
@@ -1,43 +1,55 @@
 import type { Task } from '../Task';
 import type { Grouper } from './Grouper';
-import { GroupHeadings } from './GroupHeadings';
-import { IntermediateTaskGroups } from './IntermediateTaskGroups';
+import { GroupDisplayHeadingSelector } from './GroupDisplayHeadingSelector';
+import { TaskGroupingTree } from './TaskGroupingTree';
 import { TaskGroup } from './TaskGroup';
 
 /**
- * TaskGroup stores all the groups of tasks generated by any 'group by'
+ * Store all the groups of tasks generated by any 'group by'
  * instructions in the task block.
+ *
+ * The groups are stored in array of {@link TaskGroup} objects.
+ * @see {@link Grouper}
+ * @see {@link Query.grouping}
  */
 export class TaskGroups {
     private _groups: TaskGroup[] = new Array<TaskGroup>();
     private _totalTaskCount = 0;
 
     /**
-     * Constructor for TaskGroups
-     * @param {Grouper[]} groups - 0 or more Grouping values,
+     * Constructor for {@link TaskGroups}
+     * @param {Grouper[]} groups - 0 or more {@link Grouper} objects,
```

Comment:
I think the `0 or more {@link Grouper} objects` is pretty obvious from `Grouper[]` code. What if we say that each `Grouper` represents one `group by` line in the query? Will this reflect the sense of it?